synapse/cores/postgres.py

```diff
@@ -153,3 +158,21 @@ def _initCorQueries(self):
 
     def _addVarDecor(self, name):
         return '%%(%s)s' % (name,)
+
+    def _getCoreType(self):
+        return 'postgres'
+
+    def _getBlobValuRows(self, key):
+        '''Eat specific exceptions in order to return the default value'''
+        try:
+            rows = self.select(self._q_blob_get, key=key)
+        except Exception as e:
+            if 'does not exist' in str(e):
```

Comment:
not a fan of string comparison in exception handling.  shouldn't this just return an empty set of rows?